doc/source/index.rst

```diff
@@ -6,10 +6,23 @@
 Welcome to Language Model Playground's documentation!
 =====================================================
 
+Language Model Playground is based on Pytorch. It contains several models including RNN, LSTM, GRU, Attention,
+and it can generate sentences, evaluation models, inference words. 
+Get started with :ref:`my-reference-label`.
+
+
+Language Model Playground depends on several toolkits, and the document is generated by Sphinx
+
+    * `Pytorch <https://pytorch.org/docs/stable/index.html>`_ 
```

Comment:
Use `-` to indicate bullet list.
No need for indentation.
Add description to those dependencies.
Only list import dependencies (choose wisely).
Personally I don't thinks sklearn is that important in this project.